packages/js/components/src/media-uploader/media-uploader.tsx

```diff
@@ -47,6 +48,7 @@ type MediaUploaderProps = {
 export const MediaUploader = ( {
 	allowedMediaTypes = DEFAULT_ALLOWED_MEDIA_TYPES,
 	buttonText = __( 'Choose images', 'woocommerce' ),
+	buttonProps,
```

Comment:
`buttonText` should be part of the `buttonProps`

Reply:
I don't think that's the case. buttonText is a child of the Button component, as in the following example: https://wordpress.github.io/gutenberg/?path=/docs/components-button--docs

I will leave it as it is now, if that's ok.

Reply:
it isn't :-D

`child` (actually `children`) is another component property. You could do something like the following, for instance:

```jsx

function MyCustomButton( props ) {
  return <Button children="Default button text" { ...props } />
}
```

and then, you can pass the button text (actually, it could be a component too)

```jsx
<MyCustomComponent>Hi!</MyCustomComponent>
```

or... passing the `children` prop explicitly:

```jsx
<MyCustomComponent children="'Hi!" />
```